maquetado/scss/styles.scss

```
@@ -10,10 +10,29 @@
   width: 100%;
 }
 
+.book-cover-container {
+  display: block;
+  position: relative;
+}
+
 .book-cover {
-  height: 368px;
+  height: 360px;
   margin-right: 40px;
-  width: 261px;
+}
+
+.book-badge {
+  &::after {
+    content: '';
+    background-image: url('./assets/book-badge.png');
```

You can use a shorthand for this

```suggestion
    background: url('./assets/book-badge.png') no-repeat center/contain;
```